src/main/java/io/candydoc/plugin/CandyDocMojo.java

```diff
@@ -14,7 +13,7 @@
 import org.apache.maven.project.MavenProject;
 
 @Mojo(
-    name = "candy-doc",
+    name = "ubiquitous-language",
```

Comment:
You right, we should rename the object. Tomorrow, when Candy-Doc will handle other stuff than DDD concepts, it will be interesting to decide, as a dev, which goal we want to process (ubiquitous-language, architecture, ...)